Loop54.Shared/Loop54Client.cs

```diff
@@ -110,12 +110,24 @@ public Response Sync(RequestContainer<Request> request)
         public async Task<Response> SyncAsync(RequestContainer<Request> request)
             => await CallEngineWithinClientContextAsync<Request, Response>(SyncRequestName, request, true);
 
+        public GetIndexedAttributesResponse GetIndexedAttributes(GetIndexedAttributesRequest request)
+            => GetIndexedAttributes(request.Wrap());
+        public GetIndexedAttributesResponse GetIndexedAttributes(RequestContainer<GetIndexedAttributesRequest> request)
+            => CallEngineWithinClientContext<GetIndexedAttributesRequest, GetIndexedAttributesResponse>(GetIndexedAttributesRequestName, request);
+        public async Task<GetIndexedAttributesResponse> GetIndexedAttributesAsync(GetIndexedAttributesRequest request)
+            => await GetIndexedAttributesAsync(request.Wrap());
+        public async Task<GetIndexedAttributesResponse> GetIndexedAttributesAsync(RequestContainer<GetIndexedAttributesRequest> request)
+            => await CallEngineWithinClientContextAsync<GetIndexedAttributesRequest, GetIndexedAttributesResponse>(GetIndexedAttributesRequestName, request);
+
         public Response CustomCall(string name, Request request) => CustomCall(name, request.Wrap());
         public Response CustomCall(string name, RequestContainer<Request> request) => CallEngineWithinClientContext<Request, Response>(name, request);
         public async Task<Response> CustomCallAsync(string name, Request request) => await CustomCallAsync(name, request.Wrap());
         public async Task<Response> CustomCallAsync(string name, RequestContainer<Request> request) => await CallEngineWithinClientContextAsync<Request, Response>(name, request);
 
-        private TResponse CallEngineWithinClientContext<TRequest, TResponse>(string requestName, RequestContainer<TRequest> request, bool allowNullRequest = false) where TResponse : Response where TRequest : Request
+        private TResponse CallEngineWithinClientContext<TRequest, TResponse>(string requestName, RequestContainer<TRequest> request,
```

Comment:
Nit: When splitting up a method definition on multiple lines, I prefer to have each parameter on it's own line. That makes it slightly easier to read, since you can expect all parameters to be on the same line, or each on their own line. If there's a mix you risk missing some of them when scanning the code quickly.

Reply:
Well, the downside is obviously many more lines, so you see less code on one screen. I can see the argument for "always one parameter per line" as that's easier to parse, but "one parameter per line unless they all fit on one line" seems like an odd convention to me. Of course you can miss some, but you can also miss some on one line. I don't really find it more than 2x harder to scan the full length of 2 lines than 1 line.

Reply:
Slight disagreement as if you have some parameters on separate lines, you may fool a reader into thinking they are all on separate lines and thus miss the ones that are bundled together with some others. Maybe this is just me as I try to follow the "all on one" or "all separately" rule so I get confused when someone else does not. In any case this is not blocking the PR and I'm not proposing this as a new coding guideline so I'm OK with us disagreeing here.

Reply:
> if you have some parameters on separate lines, you may fool a reader into thinking they are all on separate lines and thus miss the ones that are bundled together with some others

Yes, I suppose it depends on what the reader is used to.